docs/tables/aws_kinesis_consumer.md

```diff
@@ -0,0 +1,74 @@
+# Table: aws_kinesis_consumer
+
+A consumer, known as an Amazon Kinesis Data Streams application, is an application that you build to read and process data records from Kinesis data streams.
+
+## Examples
+
+### List all consumers
+
+```sql
+select
+  consumer_name,
+  consumer_arn,
+  consumer_status,
+  stream_arn
+from
+  aws_kinesis_consumer;
+```
+
+
+### Get a specific consumers
+
+```sql
+select
+  consumer_name,
+  consumer_arn,
+  stream_arn
+from
+  aws_kinesis_consumer
+where
+  consumer_arn = 'arn:aws:kinesis:us-east-1:986325076436:stream/turbot-data-stream/consumer/turbot-consumer:1616584220';
+```
+
+
+### List of consumers which can't read data
```

Comment:
### List of consumers which are not in ACTIVE state

```sql
select
  consumer_name,
  consumer_status,
  consumer_arn
from
  aws_kinesis_consumer
where
  consumer_status != 'ACTIVE'
```
This will help fine-grained approach for people to understand.

A consumer can't read data while in the CREATING or DELETING states.
Possible values include:
"CREATING"
"DELETING"
"ACTIVE"